saw-core/prelude/Prelude.sawcore

```diff
@@ -117,6 +119,14 @@ data Eq (t : sort 1) (x : t) : t -> sort 1 where {
     Refl : Eq t x x;
   }
 
+-- The version of Eq in Prop
+-- FIXME: this should replace the predicative Eq when we are sure everything
+-- works ok with it
+data EqP (t : sort 1) (x : t) : t -> Prop where {
+    ReflP : EqP t x x;
+  }
+
+
```

Comment:
@eddywestbrook This change of `Eq` from `sort 0` to `Prop` seems like something that would be reasonable to do in saw-core master. We should open a separate github issue requesting this change.

Reply:
Yes, this makes sense. I didn't want to change the existing definition of `Eq` as it seemed like it could be a breaking change for something and I wasn't sure how to check that it didn't break anything, but I do think it should be moved into `Prop`, just like it is in Coq.

Reply:
See #154.